src/invidious/helpers/utils.cr

```diff
@@ -10,7 +10,7 @@ def add_yt_headers(request)
   request.headers["x-youtube-client-name"] ||= "1"
   request.headers["x-youtube-client-version"] ||= "2.20200609"
   # Preserve original cookies and add new YT consent cookie for EU servers
-  request.headers["cookie"] = "#{request.headers["cookie"]?}; CONSENT=YES+"
+  request.headers["cookie"] = "#{request.headers["cookie"]?}; CONSENT=PENDING+#{Random.new.rand(900)}"
```

Comment:
Watch out, the random number should be >100 and <= 999!
[See what the peeps at youtube-dl did](https://github.com/TeamNewPipe/NewPipeExtractor/pull/600/files#diff-143031f296abdf44c2e3c0c434a9c3ed3e07db6c323b1b96aa57c6dcb618bdbeR717)

Reply:
I don't understand how that happened but it can be over 1000, it is often over 100 000

https://github.com/TeamNewPipe/NewPipeExtractor/blob/3966178979972da43f6dcbc90170058605be052b/extractor/src/test/resources/org/schabi/newpipe/extractor/kiosk/generated_mock_1.json#L10

https://github.com/TeamNewPipe/NewPipeExtractor/blob/3966178979972da43f6dcbc90170058605be052b/extractor/src/test/resources/org/schabi/newpipe/extractor/services/youtube/extractor/channel/gronkh/generated_mock_1.json#L10

and so on…

Reply:
> I don't understand how that happened but it can be over 1000, it is sometimes over 100 000

Oh, okay. I've never seen it being over 3 digits on youtube

Reply:
We should go with `Random.new.rand(100..999)` for now, so we can merge this PR and change that later, if needed...